_data/simple-icons.json

```diff
@@ -1614,6 +1614,11 @@
             "hex": "186BFF",
             "source": "https://www.autodesk.com/products/revit"
         },
+        {
+            "title": "Authentik",
+            "hex": "f05229",
+            "source": "https://github.com/Shreyask24/simple-icons/blob/develop/icons/authentik.svg"
+        },
```

Comment:
As seen in the [linter](https://github.com/simple-icons/simple-icons/actions/runs/8091532777/job/22110739938?pr=10545), you will need to position this in alphabetical order.